src/NetAnalyzers/Core/AnalyzerReleases.Unshipped.md

```diff
@@ -4,6 +4,7 @@
 
 Rule ID | Category | Severity | Notes
 --------|----------|----------|-------
+CA1311 | Globalization | Hidden | UseToLowerInvariantOrToUpperInvariant, [Documentation](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1311)
```

Comment:
Looks the link https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1311 is invalid link and maybe this is the cause of the CI failures you are seeing.

Reply:
@carlossanlop @buyaa-n could you please help with this error and have a look at the changes? I am seeing the changes are good to me.

Reply:
Thanks for replying so quickly. The link is invalid, that's true. But that makes sense for now doesn't it? I mean the analyzer itself doesn't exist yet so neither does the documentation...

Reply:
@TKharaishvili is correct. The link doesn't need to be valid (in fact will be invalid when the PR is adding a new analyzer which will be documented in future). This is not the reason for the CI failure.